### DISCUSSION TITLE
run_error_help

### DISCUSSION THREAD

**Discussion by joejiang2023** [danielver02/ALPS#154] [Q&A]
Hi all,

Could anyone please provide some help?
I can run test ICW.in successfully, but failed to run this:
mpirun -np 4 ./src/ALPS ./electron_whistler/test_whistler.in

Thanks,
-Joe

**Here is the error message:**

All processes are up and running.
Reading from Input File: test_whistler
GUESS ROUTINE: 
Intial Guess   1 :     1.0298E+00   -5.8500E-02
SPECIES PARAMETERS: 
Species   1 : 
 ns/nREF =  1.0000E+00 | qs/qREF =  1.0000E+00 | ms/mREF =  1.0000E+00
 Number of fitted functions =    1
 Relativistic effects = F
Species   2 : 
 ns/nREF =  1.0000E+00 | qs/qREF = -1.0000E+00 | ms/mREF =  5.4466E-04
 Number of fitted functions =    4
 Relativistic effects = F
-=-=-=-=-=-=-=-=-
Species  1, function  1, Maxwellian fit: 
 Initial fit parameter  1 =     2.0000E-02
 Initial fit parameter  2 =     3.2000E-01
 Initial fit parameter  3 =     0.0000E+00
 Initial fit parameter  4 =     1.0000E+00
 Initial fit parameter  5 =    -9.0000E+00
 Perpendicular correction:      1.8000E-01
Species  2, function  1, Maxwellian fit: 
 Initial fit parameter  1 =     3.0000E+03
 Initial fit parameter  2 =     1.7000E+04
 Initial fit parameter  3 =     2.0000E-02
 Initial fit parameter  4 =     1.0000E+00
 Initial fit parameter  5 =    -9.0000E+00
 Perpendicular correction:      3.0700E+03
Species  2, function  2, Maxwellian fit: 
 Initial fit parameter  1 =     1.6100E+05
 Initial fit parameter  2 =     1.7000E+04
 Initial fit parameter  3 =     1.0000E-05
 Initial fit parameter  4 =     1.0000E+00
 Initial fit parameter  5 =    -9.0000E+00
 Perpendicular correction:      1.9100E+04
Species  2, function  3, Maxwellian fit: 
 Initial fit parameter  1 =     4.6000E+03
 Initial fit parameter  2 =     2.1400E+04
 Initial fit parameter  3 =    -1.8700E-02
 Initial fit parameter  4 =     1.0000E+00
 Initial fit parameter  5 =    -9.0000E+00
 Perpendicular correction:      2.2000E+03
Species  2, function  4, bi-Moyal fit: 
 Initial fit parameter  1 =     6.0200E+03
 Initial fit parameter  2 =     1.5400E+03
 Initial fit parameter  3 =    -4.8000E-05
 Initial fit parameter  4 =     1.0000E+00
 Initial fit parameter  5 =    -9.0000E+00
 Perpendicular correction:      1.1520E+03
-=-=-=-=-=-=-=-=-
Scan 1: |k| from    5.204E-01 to    5.000E+00 at theta=   1.000E+01
-=-=-=-=-=-=-=-=-
Attempting to read f0 array from file: test_whistler
-=-=-=-=-=-=-=-=-
Calculating df0/dpperp, df0/ppar if necessary...
 Derivatives calculated
-=-=-=-=
Species   1:
 Integration:                  9.9950E-01
 Charge density:               9.9950E-01
 Parallel current density:     4.1488E-17
-=-=-=-=
Species   2:
 Integration:                  1.0000E+00
 Charge density:              -1.0000E+00
 Parallel current density:    -5.7947E-07
-=-=-=-=
 Total charge density:              -4.9874E-04
 Total parallel current density:    -5.7947E-07
-=-=-=-=-=-=-=-=
-=-=-=-=-=-=-=-=-
Determine if relativistic calculation necessary
No relativistic calculation necessary.
-=-=-=-=-=-=-=-=-
Determine fit parameters for hybrid analytic continuation...

Program received signal **SIGFPE**: Floating-point exception - erroneous arithmetic operation.

Backtrace for this error:

Could not print backtrace: executable file is not an executable
...
...
...
Could not print backtrace: executable file is not an executable
#0  0x104762afe
#1  0x104761cdd
#2  0x7fff7251e5fc
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.

**Comment by danielver02** [comment]
Hi Joe,

This is an issue with the fit parameters for the hybrid analytical continuation. The definition of the fit parameters is not quite obvious. I would recommend looking at the documentation of the code where all fit parameters are defined. In momentum space, they have different meanings than in velocity space. 

Best wishes,
-Daniel
